docs/index.rst

```diff
@@ -8,36 +8,58 @@ HPy: a better API for Python
 
 HPy provides a new API for extending Python in C.
 
+There are several advantages to write your C extension in HPy:
+
+  - **Speed**: it runs much faster on PyPy, GraalPy, and at native speed on CPython
+
+  - **Deployment**: it is possible to compile a single binary which runs unmodified on all
+    supported Python implementations and versions -- think "stable ABI" on steroids
+
+  - **Simplicity**: it is simpler and more manageable than the ``Python.h`` API, both for
+    the users and the Pythons implementing it
+
+  - **Debugging**: it provides an improved debugging experience. Debug mode can be turned
+    on at runtime without the need to recompile the extension or the Python running it.
+    HPy design is more suitable for automated checks.
```

Comment:
I shuffled this at the top, shortened the points, and added `think "stable ABI" on steroids`